includes/blocks/advanced-heading/attributes.php

```diff
@@ -12,6 +12,7 @@
 $enable_legacy_blocks = UAGB_Admin_Helper::get_admin_settings_option( 'uag_enable_legacy_blocks', ( 'yes' === get_option( 'uagb-old-user-less-than-2' ) ) ? 'yes' : 'no' );
 
 $heading_alignment_default = ( 'yes' === get_option( 'uagb-old-user-less-than-2' ) ) ? 'center' : '';
+$subheading_toggle_default = ( 'yes' === get_option( 'uagb-old-user-less-than-2' ) ) ? true : false;
```

Comment:
```suggestion
$subheading_toggle_default = 'yes' === get_option( 'uagb-old-user-less-than-2' );
```
Can we try this as we are sending same value as result of comparison

Reply:
Yes, We can do it this way also.